ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2

```diff
@@ -25,11 +53,11 @@
         </tr>
         {% for result in execution_results %}
             <tr>
-                <th>{{ result.query.title }}</th>
-                <th>{{ result.query.description }}</th>
-                <th>{{ result.query.query }}</th>
-                <th>{{ result.result }}</th>
-                <th>{{ result.error }}</th>
+                <td>{{ result.query.title }}</td>
+                <td>{{ result.query.description }}</td>
+                <td>{{ result.query.query | replace('\n', '<br>') }}</td>
+                <td class="{% if result.result == "True" %}success{% else %}error{% endif %}">{{ result.result }}</td>
```

Comment:
the result of a query might be something else than True Or False. We can have SELECT queries or ASK queries. You can leave only the {% if result.result == "True" %}success and delete the else statement

Reply:
OK